cmd/agent/main.go

```diff
@@ -55,11 +54,9 @@ func main() {
 				Namespace(metrics.NSOptions{Name: "agent"})
 
 			rOpts := new(reporter.Options).InitFromViper(v, logger)
-			tchanBuilder := tchannel.NewBuilder().InitFromViper(v, logger)
 			grpcBuilder := grpc.NewConnBuilder().InitFromViper(v)
 			builders := map[reporter.Type]app.CollectorProxyBuilder{
```

Comment:
Do we need a map here now? We have just one exporter

Reply:
It's needed for extensibility. E.g. at uber they will add custom reporters.